lib/workers/pr/index.spec.ts

```diff
@@ -410,6 +410,16 @@ describe(getName(__filename), () => {
       expect(platform.addReviewers).toHaveBeenCalledTimes(1);
       expect(platform.addReviewers.mock.calls).toMatchSnapshot();
     });
+    it('should filter assignees and reviewers based on their availability', async () => {
+      config.assignees = ['foo', 'bar'];
+      config.reviewers = ['foo', 'bar'];
+      config.filterOutUnavailableUsers = true;
+      platform.filterUnavailableUsers = jest.fn();
```

Comment:
```suggestion
```

Already mocked function

Reply:
I had that first, but Jest does not auto-mock optional functions:

```
  ● workers/pr/index › ensurePr › should filter assignees and reviewers based on their availability

    TypeError: Cannot read property 'mockResolvedValue' of undefined

      416 |       config.filterOutUnavailableUsers = true;
      417 |       // platform.filterUnavailableUsers = jest.fn();
    > 418 |       platform.filterUnavailableUsers.mockResolvedValue(['foo']);
```

Reply:
No it's because the the default mock is from github api and there it's missing. 😉

Reply:
Got it. So you're fine with the current approach?